### DISCUSSION TITLE
How to Transfer GitHub Copilot for Education to a New Account?

### DISCUSSION THREAD

**Discussion by letho1608** [community/community#146127] [GitHub Education]
### Hi everyone,

Hi everyone,

I have a situation with my GitHub accounts and would appreciate your help!

I originally had a GitHub account A registered with my school email.
I signed up for GitHub Copilot for Education on account A and enabled two-factor authentication (2FA) using an app.
Later, I changed devices and lost access to the 2FA app, so I couldn’t log in to account A.
To resolve this, I removed my school email from account A and created a new GitHub account B using the same school email.
Now, I want to use Copilot for Education on account B, but it seems that the subscription is still tied to account A.

What should I do to transfer the Copilot for Education access from account A to account B? Is there a way to release the email or reset the subscription on the new account?

Thanks in advance for your guidance!

**Comment by davevad93** [comment]
Hi @letho1608 ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) "***You cannot transfer academic discounts from one account to another. If you have more than one account you want to apply the discount to, consider merging your personal accounts and renaming the retained account if desired.***"

So it's not possible to do that because student benefits are tied to the account/email used to sign up. I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Two-factor authentication recovery` and ask for help to regain access to your old account.

**Reply by letho1608** [reply]
Thank you for your feedback, I plan to delete Github account B, but is there any way to restore access to my account A without authentication, I am still using the school email removed from account A ...

**Reply by davevad93** [reply]
Hey @letho1608 you're welcome, `2FA` is mandatory for GitHub Education so I don't think it's possible to regain access to your former account without authentication. Open a ticket as I suggested, it's the best way to find a solution for this kind of things, also keep in mind that support can take time to reply, even weeks, so be patient. 👍

**Reply by letho1608** [reply]
thank you, bro
